src/ext/BalExtension/wixlib/MbaPrereqs.wxs

```diff
@@ -0,0 +1,168 @@
+<?xml version='1.0' encoding='utf-8'?>
```

Comment:
Okay, but should we break this out into separate files? I thought @barnson did that when the Nefx.wxs got too crazy with everything in one file. Thoughts?